incubator/vault/templates/_helpers.tpl

```diff
@@ -0,0 +1,16 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "name" -}}
```

Comment:
Use namespaced templates: `vault.name` and `vault.fullname`

See #1785 and https://github.com/kubernetes/helm/pull/2845.